lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -43,6 +43,7 @@ def []=(str, value)
 end
 
 RSpec.configure do |config|
+  config.include LogStashHelper
```

Comment:
No problem, just a matter of style in here --> as I said *I tend to prefer the syntax*, it does not mean it should be like this for sure :-)